forui/lib/src/widgets/accordion.dart

```diff
@@ -0,0 +1,525 @@
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:meta/meta.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/foundation/tappable.dart';
+import 'package:forui/src/foundation/util.dart';
+
+/// A controller that stores the expanded state of an [_Item].
+interface class FAccordionController extends ChangeNotifier {
```

Comment:
In hindsight, I think we should make this a `base` class instead since an `interface` class would require them to re-implement `ChangeNotifier` too...

Reply:
If so, should `FRadioAccordionController` be a sealed class?